source/renderer/app/components/sidebar/Sidebar.js

```diff
@@ -33,16 +33,16 @@ type Props = {
 
 export type SidebarMenus = {
   wallets: ?{
-    items: Array<SidebarWalletType>,
-    activeWalletId: ?string,
-    actions: {
+    items?: Array<SidebarWalletType>,
+    activeWalletId?: ?string,
+    actions?: {
       onWalletItemClick: Function,
     },
   },
   hardwareWallets: ?{
-    items: Array<SidebarHardwareWalletType>,
-    activeWalletId: ?string,
-    actions: {
+    items?: Array<SidebarHardwareWalletType>,
+    activeWalletId?: ?string,
+    actions?: {
```

Comment:
@DeeJayElly same thing as above. Check data flow and fix issues there. It is not possible that UI will work if  you have `hardwareWallets` and not items and other parameters inside of each

Reply:
Fixed